packages/transitive-overlay/src/index.tsx

```diff
@@ -139,7 +147,9 @@ const TransitiveCanvasOverlay = ({
                       type: "Feature",
                       properties: {
                         type: "street-edge",
-                        color: modeColorMap[segment.type] || "#008",
+                        color: ignoreRouteColor
+                          ? defaultColorOverride
+                          : modeColorMap[segment.type] || "#008",
```

Comment:
Now that `defaultColorOverride` only does this, we can get rid of `ignoreRouteColor`!